src/rule-utils/index.ts

```diff
@@ -26,12 +26,19 @@ import { nodeToObject, objectHash, objectsEqual } from '../object-utils'
 import * as pointers from '../pointers'
 import { getRuleDefinition } from '../assistant'
 
+/**
+ * Object hash comparison function that ignores 'do_objectID' and '$pointer' attributes
+ */
+const adjustedObjectHash = (obj: {}, excludeKeys: string[] = []): string => {
```

Comment:
nitpick (non-blocking): Maybe we could name this in a way the describes the adjustment? Maybe `stableObjectHash`, or something?

Reply:
Nice! changing it